content/rs/administering/troubleshooting/disabling-services.md

```diff
@@ -21,11 +21,16 @@ The services that you can disable are:
 - [Discovery service]({{< relref "rs/installing-upgrading/configuring/cluster-dns.md" >}})- `mdns_server`, `pdns_server`
 - [Active-Active databases]({{< relref "/rs/administering/designing-production/active-active.md" >}}) - `crdb_coordinator`, `crdb_worker`
 
-To disable a service, use the `/v1/cluster/services/configuration` endpoint
+To disable a service with the `rladmin cluster config` command, use the `services` parameter and the name of the service, followed by `disabled`.
+```text
+ rladmin cluster config 
+        [ services <cm_server | crdb_coordinator | crdb_worker | mdns_server | pdns_server | saslauthd | stats_archiver> <enabled | disabled> ]
```

Comment:
This gets clipped [when rendered](https://docs.redislabs.com/staging/DOC-684/rs/administering/troubleshooting/disabling-services/).  Since the scrollbar doesn't appear until you interact with it, it's hard to know that there's more off-screen.  Nice-to-have: Perhaps changing to `<service>` and then listing the options separately would be a better overall experience?  (Might even consider putting them into a table with a one line description?)

Reply:
I changed it to say <service_name> because the names of the services are listed a few lines above that example.